ml-proto/spec/eval.ml

```diff
@@ -88,6 +88,9 @@ let numerics_error at = function
       error at "runtime: invalid conversion to integer"
   | exn -> raise exn
 
+let some_memory c =
+  match c.module_.memory with Some m -> m | _ -> assert false
```

Comment:
As brought up in #109, I wonder if we can stop feeding the evaluator unchecked code.  In that case, I think it'd be nice if our user of `assert` in `eval` specifically asserted "checked earlier" and so we had a clear documented distinction between actual runtime errors and things that should never happen, assuming soundness.  That is, if someone did a soundness proof, they'd prove we never hit an assert.